gpbft/vrf.go

```diff
@@ -14,19 +14,19 @@ type VRFHost interface {
 	Verifier
 }
 
-func MakeTicket(beacon []byte, instance uint64, round uint64, source PubKey, host VRFHost) (Ticket, error) {
-	return host.Sign(source, serializeSigInput(beacon, instance, round, host.NetworkName()))
+func MakeTicket(nn NetworkName, beacon []byte, instance uint64, round uint64, source PubKey, host VRFHost) (Ticket, error) {
```

Comment:
Discussion point for later date: once the number of arguments to a function go above 3 I would start to question what we might be missing in the API/interface design.

Hopefully once the implementation settles we would revisit.